src/main/java/team/gdsc/code2cv/feature/user/entity/UserAccount.java

```diff
@@ -0,0 +1,60 @@
+package team.gdsc.code2cv.feature.user.entity;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class UserAccount {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+
+	@Enumerated(EnumType.STRING)
+	private UserRole role;
+
+	private String vendorEmail;
+
+	@Enumerated(EnumType.STRING)
+	private VendorName vendorName;
+
+	private String githubToken;
+
+	private String githubNickname;
+
+	private LocalDateTime createdAt; // JPA data type LocalDateTime 추가 필요
```

Comment:
```suggestion
	private LocalDateTime createdAt;
```

LocalDateTime은 기본적으로 제공되는 타입이라 주석 없어도 될거같아요

Reply:
추후 @CreatedDate 를 추가하는 것을 고려한다는 뜻이였어요 주석의 의미가 애매했네요